composer.json

```diff
@@ -124,6 +124,11 @@
             "drush user:password editor editor",
             "drush user:unblock editor"
         ],
+        "drupal:rebuild": [
+            "sudo rm -rf vendor web composer.lock | true",
```

Comment:
It should probably also delete `patches.lock.json`.

Reply:
@phenaproxima ...even though that will lead to a dirty working directory?

Reply:
It won't. `patches.lock.json` is now gitignored in `main`.

Reply:
Oh! Well cool then; let's delete it.